extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -250,17 +250,24 @@
     "basePrefix": "?p=440,_title_,1,_screen_width_x_screen_height_,_screen_color_depth_,1,",
     "baseSuffix": ",_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id(amp-wt3-eid)_&la=_browser_language_",
     "parameterPrefix": "?p=440,_title_,1,_screen_width_x_screen_height_,_screen_color_depth_,1,_timestamp_,_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id(amp-wt3-eid)_&la=_browser_language_",
-    "parameterSuffix": "&pu=_source_url_&eor=1",
-    "pageview": "https://!trackDomain/!trackId/wt?p=440,_title_,1,_screen_width_x_screen_height_,_screen_color_depth_,1,_timestamp_,_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id(amp-wt3-eid)_&la=_browser_language_&&cp570=_page_load_time_&pu=_source_url_&eor=1",
-    "event": "https://!trackDomain/!trackId/wt?p=440,_title_,1,_screen_width_x_screen_height_,_screen_color_depth_,1,_timestamp_,_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id(amp-wt3-eid)_&la=_browser_language_&ct=webtrekk_ignore&&pu=_source_url_&eor=1",
-    "scroll": "https://!trackDomain/!trackId/wt?p=440,_title_,1,_screen_width_x_screen_height_,_screen_color_depth_,1,_timestamp_,_document_referrer_,_viewport_width_x_viewport_height_,0&tz=_timezone_&eid=_client_id(amp-wt3-eid)_&la=_browser_language_&ct=webtrekk_ignore&ck540=!verticalScrollBoundary&pu=_source_url_&eor=1",
+    "parameterSuffix": "&pu=_source_url_&_if(_if(!clientHints%252C_uach(model)___%2C%20!emptyValue), X-WT-SEC-CH-UA-MODEL, !emptyValue)=_if(_if(!clientHints%252C_uach(model)___%2C%20!emptyValue), \"_if(!clientHints%2C_uach(model)__%2C%20!emptyValue)\", !emptyValue)&_if(_if(!clientHints%252C_uach(platform)___%2C%20!emptyValue), X-WT-SEC-CH-UA-PLATFORM, !emptyValue)=_if(_if(!clientHints%252C_uach(platform)___%2C%20!emptyValue), \"_if(!clientHints%2C_uach(platform)__%2C%20!emptyValue)\", !emptyValue)&_if(_if(!clientHints%252C_uach(platformVersion)___%2C%20!emptyValue), X-WT-SEC-CH-UA-PLATFORM-VERSION, !emptyValue)=_if(_if(!clientHints%252C_uach(platformVersion)___%2C%20!emptyValue), \"_if(!clientHints%2C_uach(platformVersion)__%2C%20!emptyValue)\", !emptyValue)&eor=1",
```

Comment:
Can we simplify the tests? This is very difficult to spot what went wrong.

Reply:
I don't know how to simplify the tests. At the end there is always the complete itemized url in the tests.

Reply:
nvm, I see it's your own configuration test.